### ISSUE TITLE
[Accessibility] MAS 4.2.1: (eclipse/dialog '') An element must not have the same Name and LocalizedControlType as its parent.

### ISSUE THREAD

**Issue by soumyatippapuram** [microsoft/azure-tools-for-java#4286]
The following accessibility issue needs investigation.

[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_JavaToolingforIntelliJ/Eclipse-Benchmark; #MAS4.2.1; #JavatoolingforIntelliJ/Eclipse; #C+E-H; #Benchmark; #A11ySev2; #A11yAuto; #AI4D 

**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2020-03 (4.15.0)
Build id: 20200313-1211

**Solution Installed:** The Azure Toolkit for Eclipse 3.220, Scala IDE 4.7.x

**OS Details:**
OS: Windows (Version 1903, Build: 18362.175)

**MAS Violated:** MAS 4.2.1

**App:** eclipse
 
**Element path:** dialog ''

**Issue Details:** An element must not have the same Name and LocalizedControlType as its parent. [Section 508 502.3.1](https://www.access-board.gov/guidelines-and-standards/communications-and-it/about-the-ict-refresh/final-rule/single-file-version#502-interoperability-assistive-technology)

**How To Fix:** Provide unique names for controls that have a parent/child relationship and the same ControlType property.

This accessibility issue was found using Accessibility Insights for Windows, a tool that helps debug and find accessibility issues earlier. Get more information and download this tool at https://aka.ms/AccessibilityInsights.

**MAS Reference:**
[MAS 4.2.1](https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7BEEE97BBD-9990-48A0-8B03-6EA775581AA8%7D&file=MAS%204.2.1%20%E2%80%93%20Object%20Information.docx&action=default&mobileredirect=true)

![Bug_4286_Java Tooling_Service Principle Creation Status_AI](https://user-images.githubusercontent.com/41108985/79004032-89b3bc00-7b71-11ea-9092-f39ddabeb457.png)

**Comment by Flanker32** [comment]
@umasarath52 This issue has been fixed in #4325, you may get the [bits](https://azuredownloads.blob.core.windows.net/eclipse/accessibility/WindowsAzurePlugin4EJ-3.22.0-SNAPSHOT.develop.11968408.04-22-2020.zip) here, please help us verify it, thanks!

**Comment by soumyatippapuram** [comment]
This issue is fixed. Hence, closing this.

![Bug_4286_Java Tooling_AI_Fixed](https://user-images.githubusercontent.com/41108985/80565459-200b2d00-8a0e-11ea-8073-663da27f91a1.png)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749655
